packages/genesys-spark-components/src/components/stable/gux-tabs-advanced/gux-tab-advanced-list/gux-tab-advanced-list.scss

```diff
@@ -2,138 +2,42 @@
 @use '../../../../style/typography';
 @use '../../../../style/mixins';
 
-gux-tab-advanced-list {
-  .gux-tab-container {
+.gux-tab-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+  margin-bottom: 16px; //TODO
+  background-color: var(--gse-ui-advancedTabs-set-backgroundColor);
+
+  .gux-scrollable-section {
     display: flex;
-    align-items: flex-end;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    align-items: center;
+    justify-content: flex-start;
     width: 100%;
-    height: 50px;
-    margin-bottom: 16px;
-    overflow: auto hidden;
-    background-color: #f6f7f9;
-    border-bottom: 1px solid #d7dce5;
-    border-radius: 4px 4px 0 0;
+    overflow-x: auto;
+    border-bottom: 1px solid var(--gse-ui-advancedTabs-divider-dividerColor); //TODO
```

Comment:
I think this is the only one thats needs a token issue.

Reply:
https://inindca.atlassian.net/browse/COMUI-2444